macros/helpers/filament_swap.cfg

```diff
@@ -127,12 +133,14 @@ gcode:
         {% endif %}
     {% endif %}
 
-    {% if klippain_ercf_enabled %}
-        {% if printer.ercf.enabled %}
-            {% if printer.ercf.clog_detection > 0 %}
-                ERCF_TEST_CONFIG enable_clog_detection={printer.ercf.clog_detection}
-                {% if verbose %}
-                    RESPOND MSG="Filament loaded, ERCF clog detection reactivated"
+    {% if klippain_mmu_enabled %}
+        {% if printer.mmu.enabled %}
+            {% if printer.mmu.sync_drive is false %}
+                {% if printer.mmu.clog_detection > 0 %}
+                    MMU_TEST_CONFIG enable_clog_detection={printer.mmu.clog_detection}
+                    {% if verbose %}
+                        RESPOND MSG="Filament loaded, MMU clog detection reactivated"
+                    {% endif %}
                 {% endif %}
             {% endif %}
         {% endif %}
```

Comment:
Same here: do you think it's possible to merge this into (and use) the `_KLIPPAIN_MMU_SET_CLOGDETECTION` macro that is very similar and already used for the purge line?